wremnants/CardTool.py

```diff
@@ -286,6 +289,9 @@ def addSystematic(self, name, systAxes, outNames=None, skipEntries=None, labelsB
                 "groupFilter" : groupFilter,
                 "splitGroup" : splitGroup if len(splitGroup) else {group : ".*"}, # dummy dictionary if splitGroup=None, to allow for uniform treatment
                 "scale" : scale,
+                "sumNominal" : sumNominal,
+                "scaleHist": scaleHist,
+                "customizeNuisance" : customizeNuisance,
```

Comment:
This is a pretty generic name, can you make it more specific to whatever the use case is?

Reply:
Well, sumNominal does that for anything you want to sum to nominal, maybe "sumHistToNominal" ?
Also scaleHist -> scaleYieldsInPrefitHistogram ? I can propose something

Reply:
I mean "customizeNuisance"

Reply:
ok, right now I only needed it for scale and noConstrain, it could become more general, so I didn't call it customizeScaleAndSomethingElse. The fact is that this is already a brute force solution to allow for some flexibility in the way systematics are defined, it may also be temporary, so I am not sure how much we should care about it right now